src/System.Data.Common/tests/System.Data.Common.Tests.csproj

```diff
@@ -112,8 +112,6 @@
     <Compile Include="System\Data\VersionNotFoundException.cs" />
     <Compile Include="System\Data\XmlDataLoaderTest.cs" />
     <Compile Include="System\Data\XmlDataReaderTest.cs" />
-    <Compile Include="System\Xml\XmlDataDocumentTest.cs" />
```

Comment:
Are you planning to leave it public in the implementation? If so I would keep the tests, if not then I would either remove form the implementation or make it internal.